core/src/main/java/de/gurkenlabs/litiengine/environment/EmitterMapObjectLoader.java

```diff
@@ -47,37 +46,108 @@ public static EmitterData createEmitterData(IMapObject mapObject) {
+    data.setParticleWidth(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.STARTWIDTH_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.STARTWIDTH_MAX)));
+    data.setParticleHeight(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.STARTHEIGHT_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.STARTHEIGHT_MAX)));
+    data.setVelocityX(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.VELOCITY_X_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.VELOCITY_X_MAX)));
+    data.setVelocityY(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.VELOCITY_Y_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.VELOCITY_Y_MAX)));
+    data.setAccelerationX(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.ACCELERATION_X_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.ACCELERATION_X_MAX)));
+    data.setAccelerationY(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.ACCELERATION_Y_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.ACCELERATION_Y_MAX)));
+    data.setDeltaWidth(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTAWIDTH_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTAWIDTH_MAX)));
+    data.setDeltaHeight(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTAHEIGHT_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTAHEIGHT_MAX)));
+    data.setAngle(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.ANGLE_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.ANGLE_MAX)));
+    data.setDeltaRotation(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTA_ANGLE_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.DELTA_ANGLE_MAX)));
+    data.setParticleTTL(
+        new ParticleParameter(
+            mapObject.getFloatValue(MapObjectProperty.Particle.TTL_MIN),
+            mapObject.getFloatValue(MapObjectProperty.Particle.TTL_MAX)));
+
+    data.setCollisionType(
+        mapObject.getEnumValue(
+            MapObjectProperty.COLLISION_TYPE, Collision.class, EmitterData.DEFAULT_COLLISION));
+    data.setRequiredQuality(
+        mapObject.getEnumValue(
+            MapObjectProperty.REQUIRED_QUALITY,
+            Quality.class,
+            EmitterData.DEFAULT_REQUIRED_QUALITY));
+
+    data.setTexts(
+        mapObject.getCommaSeparatedStringValues(
+            MapObjectProperty.Particle.TEXTS, EmitterData.DEFAULT_TEXT));
```

Comment:
I believe that the maximum line length should be increased, this entire class was definitely much more readable before.